README.md

```diff
@@ -39,22 +39,30 @@ recommendations are documented at [voxpupuli.org](https://voxpupuli.org/docs/rev
 ```puppet
 class foo (
   String $bar = '',
+  String[0] $foo = '',
+  Variant $baz = '',
+  Variant[] $foobar = '',
```

Comment:
Is this even valid? Does this even compile?

Reply:
ah it's not. I remove it:
```
 $ cat test.pp
class foo (
  String $bar = '',
  String[0] $foo = '',
  Variant $baz = '',
  Variant[] $foobar = '',
) {
  # logic
}
 $ puppet parser validate test.pp
Error: Could not parse for environment production: Syntax error at ']' (file: /home/bastelfreak/code/puppet-lint_modulesync_configs/modules/voxpupuli/puppet-lint-params_empty_string-check/test.pp, line: 5, column: 11)
 $
```